shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -19,7 +19,7 @@ use derivative::Derivative;
 use futures::stream::FuturesUnordered;
 use futures::{Future, StreamExt, TryFutureExt};
 use metrics::{counter, register_counter};
-use rand::prelude::SmallRng;
+use rand::prelude::*;
```

Comment:
Why change this to the `*`?

Reply:
the idea of a prelude is it just provides the main types so that you can glob import them all.
So using a prelude without a glob import just looks wrong to me.
But I agree, lets just not use the prelude at all.

Changed.

Reply:
Yeah I was wondering if we were using more imports from there now. I'd always prefer to not use the `*` if possible.